samples/cpp/text_generation/greedy_causal_lm.cpp

```diff
@@ -9,7 +9,7 @@ int main(int argc, char* argv[]) try {
 
     std::string models_path = argv[1];
     std::string prompt = argv[2];
-    std::string device = "CPU";  // GPU can be used as well
```

Comment:
@songbell  Any particular purpose to change this? Or it is just for local debug convenience?

Reply:
sry, just local debug, will remove